src/utilities/new_versions.jl

```diff
@@ -287,7 +292,7 @@ function make_pr_for_new_version(
             @info("Commit was a success")
             api_retry() do
                 @mock git_push(
-                    "origin", new_branch_name, pkey_filename; force=true, env=env
+                    remote_name, new_branch_name, pkey_filename; force=true, env=env
```

Comment:
We actually want this "first push" to go to the HTTPS remote (not the SSH remote). That way, we can avoid triggering duplicate CI runs.